docs-2.0/2.quick-start/2.deploy-nebula-graph-with-docker-compose.md

```diff
@@ -168,6 +197,8 @@ Removing nebula-docker-compose_metad1_1    ... done
 Removing network nebula-docker-compose_nebula-net
 ```
 
+> **Note**: Command `docker-compose down -v` will **delete** all your local Nebula Graph storage data. Try this command if you're trying a developing/nightly version and having some compatible issues.
```

Comment:
Replace "trying" with "using" or "testing" would be better because the sentence starts with "Try".

Compatible issues indicate issues that are compatible. What you need might be "compatibility issues".